code/_onclick/hud/alert.dm

```diff
@@ -310,7 +310,7 @@ so as to remain in compliance with the most up-to-date laws."
 
 /obj/screen/alert/restrained/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click the alert to unbuckle unless you're handcuffed."
+	desc = "You've been buckled to something. Click the alert to unbuckle unless you're handcuffed."
```

Comment:
Ideally those cases just wouldn't throw the normal buckled alert. Of course I understand that the underlying code is all so mixed together that having such a special case is an unreasonable request right now.

Maybe make the text just say "Can't walk normally."? That should cover all cases fairly well. That might still imply "can't move" too much so just do whatever.